webapp/src/main/webapp/resources/template.soy

```diff
@@ -552,3 +552,36 @@
   <p>Please delete the parent station of the station!</p>
   <br><br>
 {/template}
+
+/**
+ * Renders a warning for routes with similar contrast to route text.
+ * @param notices : List<[routeId : string, csvRowNumber : number, routeColor : string, routeTextColor : string]>
+ */
+{template .routeColorContrast}
+{let $numNotices: length($notices)/}
+  <p class="error">Error - Route Color Contrast!</p>
+  <p>Description: The contrast between the route and the route text is too similar!</p>
+  <p><b>{$numNotices}</b> found:</p>
+  <table>
+    <thead>
+      <tr>
+        <th>Route ID</th>
+        <th>CSV Row Number</th>
+        <th>Route Color</th>
+        <th>Route Text Color</th>
+      </tr>
+    </thead>
+    <tbody>
+      {foreach $notice in $notices}
+        <tr>
+          <td>{$notice.routeId}</td>
+          <td>{$notice.csvRowNumber}</td>
+          <td>{$notice.routeColor}</td>
+          <td>{$notice.routeTextColor}</td>
```

Comment:
Can we make this more visual - for example adding a box with the correct color?

Reply:
What do you mean the 'correct color'? As in do you want me to put the colour of the route text and the route?

Reply:
As discussed, show a filled squest with the specified color.

Reply:
Done! See updated image as per comment.